packer/install-dependencies.sh

```diff
@@ -4,10 +4,14 @@ source /home/packer/provision_source.sh
 source /home/packer/cis.sh
 
 RELEASE_NOTES_FILEPATH=/var/log/azure/golden-image-install.complete
+SYSCTL_CONFIG_SRC=/home/packer/sysctl-d-60-CIS.conf
+SYSCTL_CONFIG_DEST=/etc/sysctl.d/60-CIS.conf
 
 echo "Starting build on " `date` > ${RELEASE_NOTES_FILEPATH}
 echo "Using kernel:" >> ${RELEASE_NOTES_FILEPATH}
 cat /proc/version | tee -a ${RELEASE_NOTES_FILEPATH}
+cp $SYSCTL_CONFIG $SYSCTL_CONFIG_DEST && chmod 0644 $SYSCTL_CONFIG_DEST
```

Comment:
O.K. on second thought enabling this in VHD is pretty trivial and maintainable, we still have one source of truth with the exception of file mode enforcement.

Reply:
not sure what that brings if we're already adding the file in cloud init. We should keep this pattern consistent, right now all other .conf files are written as part of cloud-init.

Reply:
It's to satisfy customers who want these configurations to not be applied "at runtime".

Reply:
Makes sense. We should do this for all the other CIS files too then (etc/issue, etc.)

Reply:
Yeah, let's use this PR as a prototype, get the patterns how we want them